Network Activity/DNS Activity/Zeek/dns_log/zeek_dns.log.event.parsed.json

```diff
@@ -0,0 +1,45 @@
+{
+    "time": 1591367999.305988,
+    "actor.user.uuid": "CMdzit1AMNsmfAIiQc",
```

Comment:
this would be `metadata.uid` based on the mapping

Reply:
Updated actor.user.uuid to reflect metadata.uid